llvm/unittests/Cheerp/CheerpPointerAnalyzerTest.cpp

```diff
@@ -36,7 +36,7 @@ TEST(CheerpTest, PointerAnalyzerTest) {
 	LLVMContext C;
 	SMDiagnostic Err;
 
-	std::unique_ptr<Module> M = parseIRFile( "test/test1.ll", Err, C );
+	std::unique_ptr<Module> M = parseIRFile( "test1.ll", Err, C );
```

Comment:
This is not correct, when the tests are fully run the path will be the existing one.

Reply:
The current path works when executed by jenkins, that is the only relevant case.

Reply:
Sure, dropping this

Reply:
Added -v to the test command: ninja check executes things from a different folder than ninja check-llvm, jenkins uses the former while I manually used the latter. Dropping this commit. The proper fix would be modifying llvm/unittests/Cheerp/CMakeLists somehow to have test.ll in the right place independently of where the command is invoked